conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -30,6 +35,44 @@ def find_packages_with_missing_files(prefix: str | Path) -> dict[str, list[str]]
     return packages_with_missing_files
 
 
+def find_altered_packages(prefix: str | Path) -> dict[str, list[str]]:
+    """Finds altered packages"""
+    altered_packages = {}
+
+    prefix = Path(prefix)
+    for file in (prefix / "conda-meta").glob("*.json"):
+        try:
+            data = json.loads(file.read_text())
+        except Exception as e:
+            raise CondaError(
+                f"Could not load the json file {file}"
+                + f" because of the following error: {e}."
+            )
+        required_data = data["paths_data"]["paths"]
+
+        for path in required_data:
+            _path = path.get("_path")
+            if _path is None:
+                continue
+            old_sha256 = path.get("sha256_in_prefix")
+            file_location = f"{prefix}/{_path}"
+            if not os.path.isfile(file_location):
+                continue
+
+            try:
+                new_sha256 = compute_sum(file_location, "sha256")
+            except OSError as err:
+                raise CondaError(
+                    f"Could not generate checksum for file {file_location}"
+                    + f" because of the following error: {err}."
```

Comment:
```suggestion
                    f"Could not generate checksum for file {file_location} "
                    f"because of the following error: {err}."
```

Reply:
thank you for explaining this, @kenodegard !